### ISSUE TITLE
about training information

### ISSUE THREAD

**Issue by Nomination-NRB** [luxiaolili/IDM-VTON-train#3]
I‘m greatful for the training code you provided, but I also have some question:

1.I wonder what the normal loss curve looks like(like start at? end at?). 
2.How many epochs should I set.
3.Is the result shown in the readme document the result of test.py or the log during training.

**Comment by luxiaolili** [comment]
> I‘m greatful for the training code you provided, but I also have some question:
> 
> 1.I wonder what the normal loss curve looks like(like start at? end at?). 2.How many epochs should I set. 3.Is the result shown in the readme document the result of test.py or the log during training.

1.The batchsize is 1, the loss is unstable. 2. I train it for 10w steps, but only 5000 steps it can realize a result not bad, You can save 10000 steps and test the result. 3. The readme document is the result of test.py

**Comment by Nomination-NRB** [comment]
> > I‘m greatful for the training code you provided, but I also have some question:
> > 1.I wonder what the normal loss curve looks like(like start at? end at?). 2.How many epochs should I set. 3.Is the result shown in the readme document the result of test.py or the log during training.
> 
> 1.The batchsize is 1, the loss is unstable. 2. I train it for 10w steps, but only 5000 steps it can realize a result not bad, You can save 10000 steps and test the result. 3. The readme document is the result of test.py

got it, thanks a lot
